packages/web/src/types.ts

```diff
@@ -236,3 +236,46 @@ export type AppCallsParam = BasicParams & AppOptionalFlags & {
 export type ASADef = z.infer<typeof ASADefSchema>;
 
 export type ASADefs = z.infer<typeof ASADefsSchema>;
+
+interface LogicSigStorageStructure {
+  tag: Buffer
+  logic: Uint8Array
+  args: Uint8Array[]
+  sig?: Uint8Array
+  msig?: EncodedMultisig
+}
+
+/** Algosdk types */
+export interface LogicSig extends LogicSigStorageStructure {
+
+  get_obj_for_encoding: () => EncodedLogicSig
+  from_obj_for_encoding: (encoded: EncodedLogicSig) => LogicSig
+
+  // Performs signature verification
+  verify: (msg: Uint8Array) => boolean
+  // Compute hash of the logic sig program (that is the same as escrow account address) as string address
+  address: () => string
+  // Creates signature (if no msig provided) or multi signature otherwise
+  sign: (secretKey: Uint8Array, msig?: MultisigMetadata) => void
+  // Signs and appends a signature
+  appendToMultisig: (secretKey: Uint8Array) => void
+  // signs and returns program signature, without appending it to this object
+  signProgram: (secretKey: Uint8Array) => Uint8Array
+  singleSignMultisig: (secretKey: Uint8Array, msig: EncodedMultisig) => [Uint8Array, number]
+  // serializes and encodes the LogicSig
+  toByte: () => Uint8Array
+  // deserializes a LogicSig which was serialized using toByte()
+  fromByte: (encoded: Uint8Array) => LogicSig
+}
```

Comment:
These are not exported in algosdk? Can we request them to export these?

Reply:
these are exported as a class, they don't have them as types. Importing class was incompatible that's why I have added this here.

Reply:
Will check once again I forgot the exact  problem

Reply:
![hk](https://user-images.githubusercontent.com/26249903/126646245-d46c1448-cba4-4714-856e-87bd1f49f2ff.png)
This was the issue.